MEAnalysis/python/MEAnalysis_heppy.py

```diff
@@ -23,6 +23,9 @@
     print "Loading ME config from", os.environ["ME_CONF"]
     meconf = imp.load_source("meconf", os.environ["ME_CONF"])
     from meconf import Conf
+elif 0:
```

Comment:
it's for crab - I would make it elif 1 when I want to run. If you know of a way to put the switch in the crab submission, that would be better.